src/local.js

```diff
@@ -1,18 +1,61 @@
 'use strict';
 
-const n = require('numeric');
 const regression = require('./regression');
 const numeric = require('numeric');
 const get = require('lodash/get');
 const fs = require('fs');
 const FreeSurfer = require('freesurfer-parser');
+const pify = require('pify');
+
+/**
+ * Add bias.
+ *
+ * @param {?} A
+ * @returns {Array}
+ */
+function addBias(arr) {
+  if (!Array.isArray(arr) || !arr.length) {
+    throw new Error(`Expected ${arr} to be an array with length`);
+  } else if (!arr.every(Array.isArray)) {
+    throw new Error(`Expected every item of ${arr} to be an array`);
+  }
+
+  return arr.map(row => row.concat(1));
+}
+
+/**
+ * Get normalized co-variate values.
+ *
+ * @todo This ignores non-numeric and non-boolean tag values. Determine
+ * an effective way to map strings and non-primitives to numbers.
+ *
+ * @param {Object} tags Hash of tag names to tag values
+ * @returns {Array}
+ */
+function getNormalizedTags(tags) {
```

Comment:
We should get our hands on some actual metadata to see what we’re dealing with. Objective and gradient calculation only accept numeric input; we’ll need to support string-to-number mapping somehow.